lib/exqlite/sqlite3.ex

```diff
@@ -303,6 +303,94 @@ defmodule Exqlite.Sqlite3 do
     Sqlite3NIF.set_log_hook(pid)
   end
 
+  @doc """
+  Binds a text value to a prepared statement.
+
+      iex> {:ok, conn} = Sqlite3.open(":memory:", [:readonly])
+      iex> {:ok, stmt} = Sqlite3.prepare(conn, "SELECT ?")
+      iex> Sqlite3.bind_text(stmt, 1, "Alice")
+      :ok
+
+  """
+  @spec bind_text(statement, non_neg_integer, String.t()) :: :ok
+  def bind_text(stmt, index, text) do
+    case Sqlite3NIF.bind_text(stmt, index, text) do
+      0 = _SQLITE_OK -> :ok
+      rc -> raise Exqlite.Error, message: errmsg(stmt) || errstr(rc)
+    end
+  end
+
+  @doc """
+  Binds a blob value to a prepared statement.
+
+      iex> {:ok, conn} = Sqlite3.open(":memory:", [:readonly])
+      iex> {:ok, stmt} = Sqlite3.prepare(conn, "SELECT ?")
+      iex> Sqlite3.bind_blob(stmt, 1, <<0, 0, 0>>)
+      :ok
+
+  """
+  @spec bind_blob(statement, non_neg_integer, binary) :: :ok
+  def bind_blob(stmt, index, blob) do
+    case Sqlite3NIF.bind_blob(stmt, index, blob) do
+      0 = _SQLITE_OK -> :ok
+      rc -> raise Exqlite.Error, message: errmsg(stmt) || errstr(rc)
+    end
+  end
+
+  @doc """
+  Binds an integer value to a prepared statement.
+
+      iex> {:ok, conn} = Sqlite3.open(":memory:", [:readonly])
+      iex> {:ok, stmt} = Sqlite3.prepare(conn, "SELECT ?")
+      iex> Sqlite3.bind_integer(stmt, 1, 42)
+      :ok
+
+  """
+  @spec bind_integer(statement, non_neg_integer, integer) :: :ok
+  def bind_integer(stmt, index, integer) do
+    case Sqlite3NIF.bind_integer(stmt, index, integer) do
+      0 = _SQLITE_OK -> :ok
```

Comment:
Linter doesn't like this 😢 besides doing something like `^@SQLITE_OK -> :ok` I think you can simply downcase it here.

Reply:
https://github.com/elixir-sqlite/exqlite/pull/298/commits/895de04a7041e32731367def3b2f5ec829fc59eb